src/main/java/com/ericsson/ei/notifications/EmailSender.java

```diff
@@ -90,6 +90,19 @@ public MimeMessage prepareEmailMessage(String recipients, String mapNotification
         return message;
     }
 
+    /*
+     * This method assigns the default values for sender and subject of email notification if values
+     * at application.proporties are empty.
+     * */
+    private void setDefaultValuesEmailSenderSubject() {
+       if (subject.isEmpty()) {
+            subject = "Email Subscription Notification";
+        }
+       if (sender.isEmpty()) {
```

Comment:
Use StringUtils.isEmpty to avoid NullPointer in these if statements

Reply:
okay